nengo_viz/static/viz_netgraph.js

```diff
@@ -104,6 +131,7 @@ VIZ.NetGraph = function(parent, args) {
             self.offsetY = (self.offsetY + yy) / scale - yy;
             
             self.scale = scale * self.scale;
+            interact.margin(30*self.scale);
```

Comment:
Hmm, this means that if you're zoomed in, the resizable margin changes.  This means that if an object is less than a certain absolute size, you can *never* drag it, even if you zoom in.